### PR TITLE
Add notify all without TO text feature

### PR DESCRIPTION
**Pull request by vigov5**

### CODE REVIEW

File: src/js/internals/NotifyAll.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                last_id = tl.getLastChatId();
+            for (let id in RM.member_dat) msg += `[To:${id}]`;
+            CS.sendMessage(RM.id, msg + token);
+            let update_timer = setInterval(() => {
+                if (tl.getLastChatId() != last_id) {
+                    window.clearInterval(update_timer);
+                    for (let i = tl.chat_list.length - 1; i > 0; i--) {
+                        if (tl.chat_list[i].msg.indexOf(token) != -1) {
+                            CS.deleteChat(tl.chat_list[i].id, null, null)
+                            CS.sendMessage(RM.id, `TO ALL >>>\n${$("#_chatText").val()}`);
```
Comment (bs90) [inline src/js/internals/NotifyAll.js]: add style for this, pls

File: src/js/helpers/Common.js [hunk 2]
```diff
@@ -179,6 +179,16 @@ class Common {
 
         return items[Math.floor(Math.random() * items.length)];
     }
+
+    random_string(n) {
```
Comment (wataridori) [inline src/js/helpers/Common.js]: Viết tên hàm dưới dạng `camelCase` cho giống với ở trên đi anh 😄

File: src/js/internals/NotifyAll.js (hunk 1 above)
Comment (bs90) [inline src/js/internals/NotifyAll.js]: https://gyazo.com/79c4b48223e47a5708b3f2902fd2db58
if possible =))

Comment (wataridori) [inline src/js/internals/NotifyAll.js]: @bs90 No way to add CSS Style here, it will be escaped on Chatwork and displayed as a HTML tag.

Comment (bs90) [inline src/js/internals/NotifyAll.js]: https://gyazo.com/2b9096778c74792d1860a7a8194f4c74
Smt like this is also impossible? (yaoming)

Comment (wataridori) [inline src/js/internals/NotifyAll.js]: @bs90 No, we can do it, nothing is impossible =)) I'll try in another commit.

File: src/js/helpers/Common.js [hunk 3]
```diff
@@ -179,6 +179,16 @@ class Common {
 
         return items[Math.floor(Math.random() * items.length)];
     }
+
+    randomString(n) {
+        let text = "";
+        let possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+        for( let i=0; i < n; i++ )
```
Comment (euclid1990) [inline src/js/helpers/Common.js]: thiếu dấu cách khi assign biến nè =))

File: src/js/internals/NotifyAll.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        $("#_notifyButton").click(() => {
+            let tl = RM.timeline,
+                last_id = tl.getLastChatId();
+            for (let id in RM.member_dat) msg += `[To:${id}]`;
+            CS.sendMessage(RM.id, msg + token);
+            let update_timer = setInterval(() => {
+                if (tl.getLastChatId() != last_id) {
+                    window.clearInterval(update_timer);
+                    for (let i = tl.chat_list.length - 1; i > 0; i--) {
+                        if (tl.chat_list[i].msg.indexOf(token) != -1) {
```
Comment (euclid1990) [inline src/js/internals/NotifyAll.js]:
```
if (~tl.chat_list[i].msg.indexOf(token))
```

**Pull request merged**